docs/documentation.html

```diff
@@ -0,0 +1,1210 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Driftcss | Documentation</title>
+		<link rel="stylesheet" href="/css/drift.css" />
+		<link rel="stylesheet" href="/css/index.css" />
+		<link rel="stylesheet" href="/css/docs.css" />
+		<link rel="stylesheet" href="/prism.css" />
+		<script defer src="/prism.js"></script>
+		<script
+			defer
+			src="https://kit.fontawesome.com/05f7314c5e.js"
+			crossorigin="anonymous"
+		></script>
+	</head>
+	<body>
+		<header>
+			<div class="container">
+				<nav class="row nav">
+					<div class="nav-left">
+						<ul class="nav-links">
+							<li class="nav-link"><a href="/">Driftcss</a></li>
+							<li class="nav-link">
+								<a href="/docs/documentation.html">Docs</a>
+							</li>
+							<li class="nav-link">
+								<a href="/docs/documentation.html">Download</a>
+							</li>
+						</ul>
+					</div>
+					<div class="nav-right">
+						<form action="/" id="docsearch">
+							<input
+								id="docsearch-input"
+								placeholder="Seach Docs..."
+								type="search"
+							/>
+						</form>
+						<ul class="nav-links social-icons hide">
+							<li class="nav-link">
+								<a href="https://github.com/duhbhavesh/driftcss"
+									><img
+										class="social-icon"
+										src="/docs/img/homepage/github.svg"
+										alt="Github"
+								/></a>
+							</li>
+							<li class="nav-link">
+								<a href="https://twitter.com"
+									><img
+										class="social-icon"
+										src="/docs/img/homepage/twitter.svg"
+										alt="Twitter"
+								/></a>
+							</li>
+						</ul>
+					</div>
+				</nav>
+				<button
+					onclick="scrollToTop()"
+					class="btn btn-square btn-primary btn-circle btn-fab btn-fab-arrow"
+				>
+					<i class="fas fa-angle-double-up"></i>
+				</button>
+			</div>
+		</header>
+		<aside class="container sidebar">

```

Comment:
sidebar is not scrollable on desktop

try adding
 
**overflow-y: auto;**

in docs.css